### DISCUSSION TITLE
choose default NIC / route

### DISCUSSION THREAD

**Discussion by danie-dejager** [amazonlinux/amazon-linux-2022#453] [General]
If I add an extra NIC to AL2023, how can I permanently update the route to use the extra/new interface. I see networking is using systemd and I'm not finding a guide that I can reliably set the default route. It works if I start it after the host came up but I'd like the default route to use ens6 and not ens5. 
![image](https://github.com/amazonlinux/amazon-linux-2023/assets/13314239/b189e9de-51f5-4fa3-b200-f91a19ba153c)

I have many Security group configurations using the IP allocated to the Extra interface (ens6). It was easy to do on AL2 with the network scripts.

**Comment by gleidsoncampos** [comment]
for anyone doing the same search, what worked for me was adding a default route through the nic that I added, with a low metric. 

something like
`ip route add default via $(ip route | grep default | grep ens5 | cut -d' ' -f3) dev ens6 metric 100`
